.ci/openshift-ci/build/Dockerfile

```diff
@@ -27,6 +27,7 @@ WORKDIR /che-plugin-registry
 RUN yum update -y && yum upgrade -y &&\
     yum install git -y &&\
     yum install openssh -y &&\
+    yum install coreutils -y &&\
```

Comment:
It would save time to do all these installs in a single step. Then you only have to hit the rpm database once.